content/blog/grpc-web-interceptor.md

```diff
@@ -209,6 +163,31 @@ TestStreamInterceptor.prototype.intercept = function(request, invoker) {
 };
 ```
 
+## Binding interceptors
```

Comment:
I updated the title because "Configuring" didn't seem quite right. If you'd prefer a term other than "Binding" let me know.